src/credentials/FabricTable.cpp

```diff
@@ -2164,4 +2164,17 @@ CHIP_ERROR FabricTable::SetFabricIndexForNextAddition(FabricIndex fabricIndex)
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR FabricTable::SetShouldAdvertiseIdentity(FabricIndex fabricIndex, AdvertiseIdentity advertiseIdentity)
```

Comment:
Is there a reason why we are recording this in the fabric table?

Reply:
That's pre-existing, but fundamentally the reason is that DnssdServer::AdvertiseOperational examines the fabric table to decide what it wants to do.